src/lib/layer.js

```diff
@@ -39,7 +39,9 @@ const defaultProps = {
   onClick: () => {},
   // Update triggers: a key change detection mechanism in deck.gl
   // See layer documentation
-  updateTriggers: {}
+  updateTriggers: {},
+  projectionMode: 1,
```

Comment:
For readability, let's use `COORDINATE_SYSTEM` enum for `projectionMode` whenever possible.